docs/Using-Fleet/Permissions.md

```diff
@@ -69,9 +69,11 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 | View Apple business manager (BM) information                                                                                               |          |           |            | ✅     |        |
 | Generate Apple mobile device management (MDM) certificate signing request (CSR)                                                            |          |           |            | ✅     |        |
 | View disk encryption key for macOS hosts enrolled in Fleet's MDM                                                                           | ✅        | ✅         | ✅          | ✅     |        |
-| Create edit and delete configuration profiles for macOS hosts enrolled in Fleet's MDM                                                      |          |           | ✅          | ✅     |        |
+| Create edit and delete configuration profiles for macOS hosts enrolled in Fleet's MDM                                                      |          |           | ✅          | ✅     | ✅      |
 | Execute MDM commands on macOS hosts enrolled in Fleet's MDM                                                                                |          |           | ✅          | ✅     |        |
 | View results of MDM commands executed on macOS hosts enrolled in Fleet's MDM                                                               | ✅        | ✅         | ✅          | ✅     |        |
+| Edit [MDM settings](https://fleetdm.com/docs/using-fleet/mdm-macos-settings)                                                               |          |           |            | ✅     | ✅      |
+| Edit [MDM settings for teams](https://fleetdm.com/docs/using-fleet/mdm-macos-settings)                                                     |          |           |            | ✅     | ✅      |
```

Comment:
The term "Edit" sounds like `Read --> modify --> write`.
I don't think this is the meaning here. This is about writing a new config by the Gitops. right?

If I am not wrong, should we rename it?

Reply:
Just trying to be consistent with existing permissions (see other rows, e.g. `Edit agent options`, etc.)

Reply:
For the sake of releasing it I am fine with keeping 'Edit'.
But in previous cases other roles also had read permission so it made sense. if it's 'Write' only then it's not 'Edit' 🙂 .

Again, I am fine with it now but let's raise it and maybe modify if needed.